### ISSUE TITLE
Add example config file for evaluating pre-trained model

### ISSUE THREAD

**Issue by nickwp** [WatChMaL/WatChMaL#3]
Add resnet_evaluate.yaml to match the resnet_train.yaml to use for loading trained weights and then evaluating the test set.

**Comment by nickwp** [comment]
This was added in #12
